homeassistant/components/cast/media_player.py

```diff
@@ -306,7 +306,7 @@ def new_media_status(self, media_status):
             url_description = ""
             if "tts" in self.hass.config.components:
                 with suppress(KeyError):  # base_url not configured
-                    tts_base_url = self.hass.components.tts.get_base_url(self.hass)
+                    tts_base_url = tts.get_base_url(self.hass)
```

Comment:
@balloob is this a case where you would recommend that I place the insert inside the function?

Reply:
Yes, because `mutagen` is a TTS requirement and might not be installed if `tts` is not in `hass.config.components`

Reply:
In the container that we distribute, we always install all requirements. This issue would have impacted only the people running Python in a virtual env. That's why our test suite won't find it.